### ISSUE TITLE
Throw error if import * as React 

### ISSUE THREAD

**Issue by bolasblack** [codemix/flow-runtime#24]
When I import react in this form:

    import * as React from "react"

babel-flow-runtime will throw error:

```
Module build failed: TypeError: /project/scripts/components/datetime_picker.jsx: Cannot read property 'name' of undefined
  at /Volumes/HDD/jimu/flux-pomotodo/node_modules/babel-plugin-flow-runtime/lib/firstPassVisitors.js:80:47
  at Array.forEach (native)
  at ImportDeclaration (/Volumes/HDD/jimu/flux-pomotodo/node_modules/babel-plugin-flow-runtime/lib/firstPassVisitors.js:69:30)
  at NodePath._call (/Volumes/HDD/jimu/flux-pomotodo/node_modules/babel-core/node_modules/babel-traverse/lib/path/context.js:76:18)
  at NodePath.call (/Volumes/HDD/jimu/flux-pomotodo/node_modules/babel-core/node_modules/babel-traverse/lib/path/context.js:48:17)
  at NodePath.visit (/Volumes/HDD/jimu/flux-pomotodo/node_modules/babel-core/node_modules/babel-traverse/lib/path/context.js:105:12)
  at TraversalContext.visitQueue (/Volumes/HDD/jimu/flux-pomotodo/node_modules/babel-core/node_modules/babel-traverse/lib/context.js:150:16)
  at TraversalContext.visitMultiple (/Volumes/HDD/jimu/flux-pomotodo/node_modules/babel-core/node_modules/babel-traverse/lib/context.js:103:17)
  at TraversalContext.visit (/Volumes/HDD/jimu/flux-pomotodo/node_modules/babel-core/node_modules/babel-traverse/lib/context.js:190:19)
  at Function.traverse.node (/Volumes/HDD/jimu/flux-pomotodo/node_modules/babel-core/node_modules/babel-traverse/lib/index.js:114:17)
  at traverse (/Volumes/HDD/jimu/flux-pomotodo/node_modules/babel-core/node_modules/babel-traverse/lib/index.js:79:12)
  at NodePath.traverse (/Volumes/HDD/jimu/flux-pomotodo/node_modules/babel-core/node_modules/babel-traverse/lib/path/index.js:144:25)
  at PluginPass.Program (/Volumes/HDD/jimu/flux-pomotodo/node_modules/babel-plugin-flow-runtime/lib/index.js:14:14)
  at newFn (/Volumes/HDD/jimu/flux-pomotodo/node_modules/babel-core/node_modules/babel-traverse/lib/visitors.js:276:21)
  at NodePath._call (/Volumes/HDD/jimu/flux-pomotodo/node_modules/babel-core/node_modules/babel-traverse/lib/path/context.js:76:18)
  at NodePath.call (/Volumes/HDD/jimu/flux-pomotodo/node_modules/babel-core/node_modules/babel-traverse/lib/path/context.js:48:17)
  at NodePath.visit (/Volumes/HDD/jimu/flux-pomotodo/node_modules/babel-core/node_modules/babel-traverse/lib/path/context.js:105:12)
  at TraversalContext.visitQueue (/Volumes/HDD/jimu/flux-pomotodo/node_modules/babel-core/node_modules/babel-traverse/lib/context.js:150:16)
  at TraversalContext.visitSingle (/Volumes/HDD/jimu/flux-pomotodo/node_modules/babel-core/node_modules/babel-traverse/lib/context.js:108:19)
  at TraversalContext.visit (/Volumes/HDD/jimu/flux-pomotodo/node_modules/babel-core/node_modules/babel-traverse/lib/context.js:192:19)
  at Function.traverse.node (/Volumes/HDD/jimu/flux-pomotodo/node_modules/babel-core/node_modules/babel-traverse/lib/index.js:114:17)
  at traverse (/Volumes/HDD/jimu/flux-pomotodo/node_modules/babel-core/node_modules/babel-traverse/lib/index.js:79:12)
  at File.transform (/Volumes/HDD/jimu/flux-pomotodo/node_modules/babel-core/lib/transformation/file/index.js:563:35)
  at /Volumes/HDD/jimu/flux-pomotodo/node_modules/babel-core/lib/transformation/pipeline.js:50:19
  at File.wrap (/Volumes/HDD/jimu/flux-pomotodo/node_modules/babel-core/lib/transformation/file/index.js:579:16)
  at Pipeline.transform (/Volumes/HDD/jimu/flux-pomotodo/node_modules/babel-core/lib/transformation/pipeline.js:47:17)
  at transpile (/Volumes/HDD/jimu/flux-pomotodo/node_modules/babel-loader/index.js:38:20)
  at Object.module.exports (/Volumes/HDD/jimu/flux-pomotodo/node_modules/babel-loader/index.js:131:12)
```

**Comment by bolasblack** [comment]
👍

**Comment by phpnode** [comment]
fixed in 0.0.7
